train.py

```diff
@@ -93,6 +93,7 @@ def train_model(args, device, parallel):
     # make data loader
     ds_cfgs = [a for a in sorted(args.keys()) if 'dataset' in a]
```

Comment:
Not so sure what this line does. Are there more than one 'dataset' keys expected?

Reply:
This is for mixed batch ratio experiments where you can specify the batch size for multiple datasets